### ISSUE TITLE
An Option to Make the Notification Sticky

### ISSUE THREAD

**Issue by KaiserCalm** [CorruptedArk/did-i-take-my-meds#65]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I clear the notification by accident.

**Describe the solution you'd like**
Adding an option to make the notification be unclearable until the medication is taken.

**Comment by rociomarielgnas** [comment]
Agreed. Or have it pester you if no action has been taken after a set amount of time. 
Today I absentmindedly cleared all my notifications and got distracted with work and forgot to take my pills until now. It would be really helpful to have it notifiy me again after an hour or so because I didn't press anything on the notif.

**Comment by gammeredith** [comment]
Agreed.
